.github/workflows/ci.yml

```diff
@@ -2,6 +2,7 @@ name: CI
 
 on:
   workflow_dispatch:
+  merge_group:
```

Comment:
I think we need to update the concurrency group?

Reply:
I'm not sure, but feel free to edit the PR if you know exactly :)

Reply:
I'm not sure either, seems like the PR currently works fine

Reply:
I thought we need this because of the cancelled ci runs